main.go

```diff
@@ -73,6 +73,7 @@ func saveDone(filename, title string) {
 
 func main() {
 	var indexPath string
+	var indexPath2 string
```

Comment:
**[errcheck]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
 indexPath2 declared but not used